### PR TITLE
Feature/cdap 648 use wait for status

### PR DESCRIPTION
**Pull request by ParthGandhi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -76,6 +76,12 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
+      <groupId>com.google.inject</groupId>
```
Comment (albertshau) [inline pom.xml:79]: why is guice needed?

Comment (ParthGandhi) [inline pom.xml:79]: This app fails to even compile. It uses `Types` here: https://github.com/cdap-guides/cdap-mapreduce-guide/blob/feature/CDAP-648-useWaitForStatus/src/main/java/co/cdap/guides/LogAnalyticsApp.java#L25 which comes from Guice.

Comment (albertshau) [inline pom.xml:79]: ogosh

**Comment by albertshau** [comment]
:+1:

**Pull request merged**